google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineAssetProvider.java

```diff
@@ -44,4 +48,20 @@ public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Ar
 
     return null;
   }
+
+  @Nullable
+  @Override
+  public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Module module) {
+    Collection<VirtualFile> appEngineWebXmls
+        = FilenameIndex.getVirtualFilesByName(
+            project, "appengine-web.xml", module.getModuleContentScope());
+
+    for (VirtualFile contentRoot : appEngineWebXmls) {
```

Comment:
the name was a leftover before refactoring. I'll update it.

Good points about making the appengine-web.xml more robust. I didn't think multiple configs was something we needed to worry about (since its an invalid setup). But logging warnings and having priority is a good idea.